src/frontend/packages/lib_video/src/utils/resumeLive.ts

```diff
@@ -1,41 +1,38 @@
-import { getResource, modelName, Video } from 'lib-components';
+import { fetchWrapper } from 'lib-components';
 // module responsible to fetch an HLS manifest and detect when tag
 // EXT-X-ENDLIST has been removed in it.
 import { Parser } from 'm3u8-parser';
 
 const regex = /^(https.*)\/.*\.m3u8$/;
 
 const fetchManifest = async (manifestUrl: string) => {
-  const manifest = await fetch(manifestUrl).then((response) => response.text());
+  const manifest = await fetchWrapper(manifestUrl).then((response) =>
```

Comment:
I think this is not needed. There is no authentication on this endpoint.

Reply:
I find it quite overkill too but fetchWrapper was used to poll the hls.
Do you confirm I only fetch here ?

Reply:
it's useless to use it, no jwt token is use here